src/elements/teaser-product/teaser-product-static/readme.md

```diff
@@ -5,7 +5,9 @@ otherwise, see [sbb-teaser-product](/docs/elements-sbb-teaser-sbb-teaser-product
 
 ```html
 <sbb-teaser-product-static>
-  <sbb-image slot="image" image-src="..."></sbb-image>
+  <figure slot="image" class="sbb-figure">
```

Comment:
Like the other comments, here a figure is not needed. maybe make two examples, one with and one without a figure? Applies to other examples in this readme.